### PR TITLE
Modify the translated PII files in nls folder 20230605

### PR DESCRIPTION
**Pull request by bjcdong**
Files update for PII_Ship1.

Signed-off-by: Dong Chen <bjcdong@cn.ibm.com>

### CODE REVIEW

File: runtime/nls/j9cl/j9jcl_ca.nls [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # [1] https://www.gnu.org/software/classpath/license.html
 # [2] https://openjdk.org/legal/assembly-exception.html
 #
-# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
```
[the remark this answers is not in the document]
Comment (pshipton) [reply runtime/nls/j9cl/j9jcl_ca.nls:20]: This applies to all of the modified files, not just this one.

**Comment by pshipton** [comment]
jenkins compile win jdk17

**Comment by pshipton** [comment]
Regardless of the SPDX-License-Identifier problem which blocks the merge, I've created a new Windows build that can be used for testing.
https://openj9-artifactory.osuosl.org/artifactory/ci-openj9/Build_JDK17_x86-64_windows_Personal/693/OpenJ9-JDK17-x86-64_windows-20230605-110855.tar.gz

**Comment by bjcdong** [comment]
Close this one.

**Pull request closed without merging**